WhisperAPI/WhisperAPI/Services/SuggestionsService.cs

```diff
@@ -1,18 +1,38 @@
 ﻿using System.Collections.Generic;
 using WhisperAPI.Models;
+using WhisperAPI.Models.NLPAPI;
 
 namespace WhisperAPI.Services
 {
     public class SuggestionsService : ISuggestionsService
     {
         private readonly IIndexSearch _indexSearch;
 
-        public SuggestionsService(IIndexSearch indexSearch)
+        private readonly INlpCall _nlpCall;
+
+        private readonly List<string> _intents;
```

Comment:
- Not clear enough what ` List<string> _intents` is used for (are they relevant intent or intent that we want to discard?)

Reply:
Done